docs/src/main/asciidoc/rest-client.adoc

```diff
@@ -296,7 +296,7 @@ public interface Client {
 }
 ----
 
-More information about this annotation can be found on the javadoc of link:https://javadoc.io/doc/io.quarkus/quarkus-rest-client/latest/io/quarkus/rest/client/reactive/ClientFormParam.html[`@ClientFormParam`].
+More information about this annotation can be found on the javadoc of link:https://javadoc.io/doc/io.quarkus/quarkus-resteasy-client/latest/io/quarkus/rest/client/reactive/ClientFormParam.html[`@ClientFormParam`].
```

Comment:
Both of these URLs return Error code 521

Reply:
Yeah javadoc.io appears to be down. I don't know if it's temporary or completely dead.